docs/repo/acquisition/optional-workloads.md

```diff
@@ -0,0 +1,18 @@
+Add Support for Optional .NET Workloads
```

Comment:
This file is outdated, I'm still updating it with details of the current design.